### PR TITLE
Update power docs with info on how to calibrate

### PR DESCRIPTION
**Pull request by hamishwillee**
This is supposed to address #119.

@DonLakeFlyer @sanderux  I have not actually done battery setup before so this needs a review. As I understand it (and what this says) is that you enter battery cells, and high/low voltage values and the other values are auto-generated. However you can do further calibration by measuring voltage/current and entering these values to recalculate the voltage divider and volts/amp.

Is this correct?

### CODE REVIEW

**Comment by sanderux** [comment]
The other values are not auto generated. You always need to do a calibration of both the voltage and the current divider. Sometimes these are provided with the power module or vehicle.

**Comment by hamishwillee** [comment]
> Sometimes these are provided with the power module or vehicle.

Thanks @sanderux  -what do you mean "provided by the vehicle"? Do you mean that the power module will supply all the values?

**Comment by sanderux** [comment]
I mean the supplier sometimes provides the numbers. Or if you buy a vehicle that is running something like APM you could get the values from that system.

**Comment by hamishwillee** [comment]
Thanks. I've updated accordingly. It does not talk about the ardupilot "automated" update of values, but could do so if @DonLakeFlyer wants to comment on how "exactly" that works. 

OK for this to be merged?

File: en/SetupView/Power.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+![Battery Calibration](../../images/setup/PX4Power.jpg)
+
+## Battery Voltage/Current Calibration
+
+Enter data for your battery/power module from its data sheet: number of cells, full voltage per cell, empty voltage per cell. If provided, also enter voltage divider and amps-per-volt information. 
+
+*QGroundControl* can be used to calculate appropriate voltage divider and amps-per-volt values from measurements:
+
+1. Measure the voltage from the battery using a multimeter.
+1. Click **Calibrate** next to the *Voltage divider* field. On the prompt that appears:
```
Comment (DonLakeFlyer) [inline en/SetupView/Power.md]: It's Calculate not Calibrate. Same below

**Comment by DonLakeFlyer** [comment]
> Or if you buy a vehicle that is running something like APM you could get the values from that system.

Can someone point me to how that works? Wondering if there is some missing QGC feature there.

**Comment by sanderux** [comment]
Suppose you buy a Solo, it comes with APM and already has the calibration values. no need to get fancy current measurement tools to calibrate if you want to switch to px4, just copy the dividers.

**Comment by DonLakeFlyer** [comment]
Oh. I thought you were saying that there was some ArduPilot coded feature which you could call to automatically set these or something.

**Comment by sanderux** [comment]
No, but i do have a much better way to calibrate the current sensor.
It uses 2 readings, a low and a high current reading.
Based on that it can determine not only the scaling but also the offset.

[cscalib.py.zip](https://github.com/mavlink/qgc-user-guide/files/1545244/cscalib.py.zip)

File: en/SetupView/Power.md (hunk 1 above)
Comment (hamishwillee) [reply en/SetupView/Power.md]: Thanks. Fixed

**Comment by hamishwillee** [comment]
> Oh. I thought you were saying that there was some ArduPilot coded feature which you could call to automatically set these or something.

Me too!

> No, but i do have a much better way to calibrate the current sensor. It uses 2 readings, a low and a high current reading. Based on that it can determine not only the scaling but also the offset.

I looked at that and it made sense. Worth getting into QGC @DonLakeFlyer

**Comment by hamishwillee** [comment]
@DonLakeFlyer I fixed that calibrate/calculate typo. Can this be merged now?

**Comment by DonLakeFlyer** [comment]
yup

**Comment by hamishwillee** [comment]
Thanks, merged. 
FYI, I also created https://github.com/mavlink/qgroundcontrol/issues/5877 to track @sanderux current sensor calibration approach.

**Pull request merged**